examples/specialty_plots/ishikawa.py

```diff
@@ -159,78 +159,73 @@ def ishikawaplot(data,figSize=(20,10),left_margin:float=0.05,right_margin:float=
-    
-    
-    plt.tight_layout()   
+    ax.text((1 - right_margin - pd_width), 0.5, str.upper(list(data.keys())[0]), fontsize=12, weight='bold',
+            verticalalignment='center', horizontalalignment='left', bbox=props)
+
+    # draw fish tail
+    x = (0.0, 0.0, left_margin)
+    y = (0.5 - left_margin, 0.5 + left_margin, 0.5)
+    ax.fill(x, y, color=spine_color)
+
+    # draw spines with recursion
+    drawspines(data=data[list(data.keys())[0]], ax=ax, parentspine=main_spine, alpha_ps=alpha_ps, alpha_ss=alpha_ss,
+               primary_spines_rel_xpos=primary_spines_rel_xpos, spine_color=spine_color)
+
+    plt.tight_layout()
     return fig
 
+
 # =============================================================================
 # Rooting
 # =============================================================================
 current_dir = Path(__file__).parent
 output_path = current_dir / "Ishikawa.jpeg"
 
-
 # =============================================================================
 # USER DATA
 # =============================================================================
-data = {'problem': {'machine': {'cause1':''},
-                    'process': {'cause2': {'subcause1':'','subcause2':''}
-                               },
-                    'man':     {'cause3': {'subcause3':'','subcause4':''}
-                               },
-                    'design':  {'cause4': {'subcause5':'','subcause6':''}
-                                }
-                    
-                   }
-       }
+data = {'problem': {'machine': {'cause1': ''},
+                    'process': {'cause2': {'subcause1': '', 'subcause2': ''}
+                                },
+                    'man': {'cause3': {'subcause3': '', 'subcause4': ''}
+                            },
+                    'design': {'cause4': {'subcause5': '', 'subcause6': ''}
+                               }
+
+                    }
+        }
 
 
-# =============================================================================
-# MAIN CALL
-# =============================================================================
 if __name__ == '__main__':
-    
-    #try also without opt primary_spines rel_xpos
-    fig = ishikawaplot(data,figSize=(20,10),primary_spines_rel_xpos=[0.8,0.7,0.6,0.4])
-    
+    # try also without opt primary_spines rel_xpos
+    fig = ishikawaplot(data, figsize=(20, 10), primary_spines_rel_xpos=[0.8, 0.7, 0.6, 0.4])
+
     fig.show()
-    
-    fig.savefig(output_path, 
-                     #dpi=800, 
-                     format=None, 
-                     #metadata=None,
-                     bbox_inches=None, 
-                     pad_inches=0.0,
-                     facecolor='auto', 
-                     edgecolor='auto',
-                     orientation='landscape',
-                     transparent=False,
-                     backend=None)
-    
\ No newline at end of file
+
+    fig.savefig(output_path,
```

Comment:
Please do not add savefig calls in examples.  Don't need show, and probably don't need to encapsulate in a `__main__`.

Reply:
To elaborate a bit more, the way our documentation build system works is we run the examples and then find and show any figures that are created.

While using `'__main__'` is a good practice for stand-alone scripts, will effectively hide the example from the docs which will in turn prevent the rendered image from showing up in the build docs!